.eslintrc.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     env: {
       jest: true,
     },
+    rules: {
+      'jsx-a11y/label-has-for': 'off',
```

Comment:
This is being deprecated in favor of: label-has-associated-control. 

Is that why we are turning it off here?

Reply:
No but that's a good point.

I am disabling the a11y rule for test files. When we write inline React in our tests for example / test code, we don't really care about setting up the "perfect" a11y component since we're testing particular aspects of a component. I turned it off so we don't waste time with test data.